### PR TITLE
Feature/myst 111 API End Point for saving identity stats

### PR DESCRIPTION
**Pull request by interro**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -189,6 +189,24 @@ def client_send_stats():
     })
 
 
+# End Point to save identity
+@app.route('/v1/save_identity', methods=['POST'])
```
Comment (Waldz) [inline app.py]: POST /v1/identities/

File: app.py [hunk 2]
```diff
@@ -189,6 +189,24 @@ def client_send_stats():
...
+@validate_json
+def save_identity():
+    payload = request.get_json(force=True)
+
+    identity_arg = payload.get('identity', '')
```
Comment (Waldz) [inline app.py]: `identity` -> `id`

File: tests/test_save_identity.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import requests
+import json
+
+def test_save_identity():
```
Comment (Waldz) [inline tests/test_save_identity.py]: +1

**Comment by Waldz** [comment]
Rename PR to have format: "MYST-111 Idea of my change"

File: app.py (hunk 2 above)
Comment (interro) [reply app.py]: Yes, it accepts any fields

File: app.py (hunk 1 above)
Comment (interro) [reply app.py]: Will rename it

**Pull request merged**